modules/xml/test/src/smithy4s/xml/XmlCodecSpec.scala

```diff
@@ -30,10 +30,11 @@ import smithy4s.Blob
 import smithy4s.Document
 import smithy4s.Hints
 import smithy4s.ShapeId
+import smithy4s.example.OpenEnumTest
+import smithy4s.example.OpenIntEnumTest
 import smithy4s.schema.Schema
 import smithy4s.schema.Schema._
 import weaver._
-import smithy4s.example.{OpenEnumTest, OpenIntEnumTest}
```

Comment:
Let's add test cases for the new primitives to this XmlCodecSpec too since the others are there already